src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -586,29 +600,33 @@
       },
 
       async newLinkTab() {
-        if (this.amountForCredit > 0) {
+        if (this.amountForCredit <= 0) {
+          this.errors.credit.amount = "Le montant à créditer doit être un nombre positif"
+          return
+        }
+        try {
           if (!this.selectedCreditAccount) {
             if (this.creditableMoneyAccounts.length > 1) {
               throw new Error("Unexpected multiple creditable account found.")
             }
             this.selectedCreditAccount = this.creditableMoneyAccounts[0]
           }
-          try {
-            let url = await this.selectedCreditAccount._obj.getCreditUrl(
-              this.amountForCredit
-            )
-            this.urlForHyperlink = url.order_url
-          } catch (error) {
-            console.log(error)
-            this.$Swal.fire({
+          let url = await this.selectedCreditAccount._obj.getCreditUrl(
+            this.amountForCredit
+          )
+          this.urlForHyperlink = url.order_url
+          this.errors.credit.amount = false
+        } catch (err) {
+          console.log('Payment failed:', err)
+          this.$toast.error(
```

Comment:
One of my last PRs was a proposal to enforce the same toast notification system everywhere (PR https://github.com/Lokavaluto/monujo/pull/220), you should use it instead

Reply:
@SeddikKadi Your PR #220 is not validated nor merged yet because we are waiting for a UI change in #216 on your part. Can you address the issue please ?